plugins/woocommerce/changelog/fix-43617

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: enhancement
+
+Improve hover style on product tabs
```

Comment:
What do you think about updating the changelog to mention this change only affects the _Minimal_ style?

```suggestion
Improve hover style on product tabs when using the Minimal style in the Product Details block
```